src/main/groovy/grails/plugins/airbrake/AirbrakeAppender.groovy

```diff
@@ -17,8 +21,8 @@ class AirbrakeAppender<E> extends AppenderBase<E> {
 
     @Override
     protected void append(E event) {
-        if ((event?.throwableInformation || includeEventsWithoutExceptions) ) {
-            notifier.notify(event.throwableInformation?.throwable, [errorMessage: event.message.toString()])
+        if ((event?.getThrowableProxy()?.getThrowable() || includeEventsWithoutExceptions)) {
+            notifier.notify(event.getThrowableProxy().getThrowable(), [errorMessage: event.getMessage()])
```

Comment:
It seems that event is not of the same class type as in previous grails versions.
So i looked here: http://logback.qos.ch/apidocs/ch/qos/logback/classic/spi/ThrowableProxy.html to find what i should use. And getMessage returns already a String